glpk/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export CFLAGS="-fPIC -O3"
+./configure --prefix=$PREFIX --with-pic
```

Comment:
It's there because I have had errors with old versions not linking
correctly with distutils. I'll test without it to see if that's still the
case.
On Nov 21, 2015 11:40 AM, "Carlos Cordoba" <notifications@github.com> wrote:

> In glpk/build.sh
> <https://github.com/conda/conda-recipes/pull/481#discussion_r45549823>:
>
> > @@ -0,0 +1,5 @@
> > +#!/bin/bash
> > +
> > +export CFLAGS="-fPIC -O3"
> > +./configure --prefix=$PREFIX --with-pic
>
> Why fpic and pic?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-recipes/pull/481/files#r45549823>.
>